server/app.py

```diff
@@ -1,16 +1,23 @@
 import os
 from flask import Flask
 from dotenv import load_dotenv
+from flask_bcrypt import Bycrypt
 from account import account # import the account blueprint object
```

Comment:
I think we need to group these kinds of comments like 
```python
# blueprints
from account import account
```

Because if we import other things we need to write almost the same comment
For example
```python
from account import account # import the account blueprint object
from auth import auth # import the auth blueprint object
```